sources/packages/backend/libs/services/src/students-assessments/assessment-sequential-processing.service.ts

```diff
@@ -343,4 +385,139 @@ export class AssessmentSequentialProcessingService {
       options?.alternativeReferenceDate,
     );
   }
+
+  /**
+   * Get SFAS application awards totals.
+   * @param assessment current assessment being processed.
+   * @param firstAssessmentDate date of the first assessment date of the current application.
+   * @returns SFAS application awards totals.
+   */
+  async getProgramYearSFASAwardsTotals(
```

Comment:
If there is no intent to use it externally please have it declared as private, same for `getProgramYearSAILAwardsTotals`.